vault/login_mfa.go

```diff
@@ -1990,7 +1996,7 @@ func (c *Core) validateOkta(ctx context.Context, mConfig *mfa.Config, username s
 		return err
 	}
 
-	ctx, client, err := okta.NewClient(ctx,
+	ctx, oktaClient, err := okta.NewClient(ctx,
```

Comment:
Avoids shadowing `client` in `vault/identity_store_oidc_provider.go`:
https://github.com/hashicorp/vault/blob/58a5b77bc6f0a5ac47e3e3d6020148b585f7213a/vault/identity_store_oidc_provider.go#L95